mysql-test/suite/galera/t/galera_sst_mysqldump.cnf

```diff
@@ -5,7 +5,5 @@
 
 [mysqld.1]
 wsrep_provider_options='base_port=@mysqld.1.#galera_port;gcache.size=1;pc.ignore_sb=true'
-wsrep_sync_wait=0
```

Comment:
WSREP_SYNC_WAIT_NONE= 0x0  is a default so this will not have desired effect, maybe you mean this to be changed to  wsrep_sync_wait=15  ?

Reply:
In galera_2nodes.cnf for each node it set wsrep_sync_wait=15